manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster9

```

```suggestion
PackageIdentifier: IObit.DriverBooster10
```
You'll also need to change `manifests/i/IObit/DriverBooster9` to `manifests/i/IObit/DriverBooster10` and update the file names as well.

---

if someone has v9 it would receive v10 with Winget upgrade?